manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/how-to-configure-alerts.md

```diff
@@ -111,7 +115,45 @@ data0
 > **Note**:
 > - The `noRecoveries()` method suppresses recovery alerts, ensuring only critical alerts are sent.
 
+### Subscribing to OPC UA Alerts using Sample OPCUA Subscriber
+
+1. Expose the port `4840` of `ia-opcua-server` service in [docker-compose.yml](https://github.com/open-edge-platform/edge-ai-suites/blob/main/manufacturing-ai-suite/wind-turbine-anomaly-detection/docker-compose.yml#L183) as below
+
+    ```yaml
+    ia-opcua-server:
+    ...
+    ports:
+    - "4840:4840"
+    ```
+
+2. Deploy the Sample App using `make up_opcua_ingestion`
```

Comment:
We can update this as below:
2. Deploy the sample app by following below commands:

```bash
cd edge-ai-suites/manufacturing-ai-suite/wind-turbine-anomaly-detection 
make up_opcua_ingestion
```